### PR TITLE
Clean up gemspec

### PR DESCRIPTION
**Pull request by princemaple**
:koala:

### CODE REVIEW

**Comment by princemaple** [comment]
Ready now
@zendesk/quokka

File: zendesk_apps_support.gemspec [hunk 1]
```diff
@@ -17,10 +17,10 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'json-stream'
   s.add_runtime_dependency 'image_size'
   s.add_runtime_dependency 'erubis'
-  s.add_runtime_dependency 'jshintrb',    '0.2.4'
+  s.add_runtime_dependency 'jshintrb'
```
Comment (sandlerr) [inline zendesk_apps_support.gemspec]: why? We don't want people using older versions of jshintrb, that would fail some of our official apps.

**Comment by princemaple** [comment]
@sandlerr right

**Comment by sandlerr** [comment]
+1

**Comment by atroche** [comment]
:+1:

**Pull request merged**